npm/css-src/sass/_drag-and-drop.scss

```diff
@@ -2,31 +2,55 @@
   margin-bottom: 40px;
 }
 
+.drag-and-drop__button {
+  margin-bottom: 32px;
+  margin-top: 65px;
+  padding-left: 15px;
+  position: static;
+  width: auto;
+}
+
+.drag-and-drop__button::before {
+  position: static;
+}
+
+.drag-and-drop__button--highlight {
+  background-color: $govuk-focus-colour;
+  box-shadow: 0 -2px $govuk-focus-colour, 0 4px govuk-colour('black');
+  color: $govuk-text-colour;
+  outline: none;
+}
+
 .drag-and-drop__dropzone {
   display: flex;
   margin-bottom: 20px;
   outline: 3px dashed $govuk-input-border-colour;
-  padding: 60px 15px;
+  padding-bottom: 30px;
+  padding-top: 0;
   text-align: center;
   transition: outline-offset 0.1s ease-in-out, background-color 0.1s linear;
 }
 
-.drag-and-drop__input {
-  display: none;
-  left: 10.1em;
-  position: relative;
+.drag-and-drop__dropzone--dragover {
+  background: govuk-colour('mid-grey');
+  outline-color: govuk-colour('dark-grey');
 }
 
 .drag-and-drop__hint-text {
   margin-bottom: 0;
   margin-right: 10px;
-  padding-top: 7px;
+  padding-left: 15px;
+  padding-top: 70px;
 }
 
-.drag-and-drop__button {
-  display: inline-block;
-  margin-bottom: 0;
-  width: auto;
+.drag-and-drop__input {
+  /*Since input fields allow you to drag and drop folders, we've decided to make
+   the input as large as the dropzone box to avoid dealing with drop events*/
+  height: 166px;
+  opacity: 0; /*Opacity is 0 in order to keep hide the input button AND keep it functioning ('visibility' and 'display' properties can't do this)*/
```

Comment:
Should "keep hide the input button" be "keep the input button hidden"?

Reply:
Ah :man_facepalming: . Good spot! I think I restructured my sentence and forgot to delete the word "keep". It was meant to be "in order to hide the input button". Thanks, I'll fix that.